nominatim/nominatim.py

```diff
@@ -124,6 +124,15 @@ def connect(self):
         self.wait()
 
         self.cursor = self.conn.cursor()
+        # Disable JIT and parallel workers as they are known to cause problems.
+        # Update pg_settings instead of using SET because it does not yield
+        # errors on older versions of Postgres where the settings are not
+        # implemented.
+        self.perform(
+            """ UPDATE pg_settings SET setting = -1 WHERE name = 'jit_above_cost';
+                UPDATE pg_settings SET setting = 0 
+                   WHERE name = 'max_parallel_workers_per_gather';""")
+        self.wait()
```

Comment:
@lonvia instead of performing UPDATE on pg_settings you can do similar stuff thanks to `set_config`:
```sql
SELECT set_config('max_parallel_workers_per_gather','0',false) from pg_settings where name = 'max_parallel_workers_per_gather';
```

Reply:
This was intentionally done this way. UPDATE conveniently does nothing on older Postgresql versions where the settings do not exist. The alternative would have been to check for the Postgresql version first and that's just painful.